rpcs3/Emu/RSX/Overlays/overlay_trophy_notification.cpp

```diff
@@ -133,11 +135,11 @@ namespace rsx
 			text_view.auto_resize();
 
 			// Resize background to cover the text
-			u16 margin_sz = text_view.x - image.w - image.x;
-			frame.w       = text_view.x + text_view.w + margin_sz;
+			u16 margin_sz = 9;
```

Comment:
I think using this margin is fine. But it should probably be used to calculate the image position etc as well for consistency

Reply:
Note: Removal of auto resize means lots of testing specifically with different languages that dont use latin fonts. This will be needed with and without fw and also on linux and BSD. This hardcoded stuff to me just seems to be whats appealing in specific conditions. We had ugly overflowing text when using some system languages before.

Reply:
Tested Portuguese, English US, Japanese, Traditional Chinese, Korean, Russian and Danish. 18 different trophies each test. The only problem I noticed is that the first line remains in English, but this already happens in the current build so it isn't my fault.